cognite/client/data_classes/files.py

```diff
@@ -447,6 +482,18 @@ def security_categories(self) -> _ListFileMetadataUpdate:
 
     @classmethod
     def _get_update_properties(cls, item: CogniteResource | None = None) -> list[PropertySpec]:
+        if isinstance(item, (FileMetadata, FileMetadataWrite)) and item.instance_id:
+            return [
+                # If Instance ID is set, the file was created in DMS. Then, it is
+                # limited which properties can be updated. (Only the ones that are not in DMS + security categories)
```

Comment:
Maybe update the comment? No security categories here

Reply:
`Only the ones that are not in DMS + security categories` I guess this is confusing? It mentions that security categories is not supported.